src/main/kotlin/com/sourcegraph/cody/commands/CommandId.kt

```diff
@@ -1,7 +1,7 @@
 package com.sourcegraph.cody.commands
 
-enum class CommandId(val displayName: String) {
-  Explain("Explain Code"),
-  Smell("Smell Code"),
-  Test("Generate Test")
+enum class CommandId(val displayName: String, val mnemonic: Char) {
+  Explain("Explain Code", 'E'),
+  Smell("Smell Code", 'S'),
+  Test("Generate Test", 'T')
```

Comment:
Great idea with those mnemonic keys!

BTW I see a potential UX issue: 

* Mnemonic combination is Alt + E, S, or T
* Shortcut version is Alt + Shift + 1, 2, or 3

IMO we should pick one and use the same keys. If you're not sure which ones to choose, it seems to me that your suggestion (E, S, and T) is definitely a better version to to memorize (+ it is compatible with mnemonic letters).

Reply:
Please see my comment above.

tl;dr
Only letters that are part of the action texts can be used for mnemonics. Mnemonics is not the same thing as shortcuts.